src/libtrx/include/libtrx/game/output/const.h

```diff
@@ -1,14 +1,8 @@
 #pragma once
 
-#if TR_VERSION == 1
-    #define MAX_OBJECT_TEXTURES 8192
-    #define MAX_SPRITE_TEXTURES 1024
-    #define MAX_TEXTURE_PAGES 128
-#else
-    #define MAX_OBJECT_TEXTURES 2048
-    #define MAX_SPRITE_TEXTURES 512
-    #define MAX_TEXTURE_PAGES 32
-#endif
+#define MAX_OBJECT_TEXTURES 8192
+#define MAX_SPRITE_TEXTURES 1024
```

Comment:
Do we still need these limits, considering we now allocate these dynamically?

Reply:
Thank you, great point. I've removed the limits for the texture infos; I'll look at potentially doing the same for pages in the next PR. I updated the lab texture array too here to use game_buf as well.